framework/Models/PostProcessors/Validations/Representativity.py

```diff
@@ -0,0 +1,272 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+  Created on April 29, 2021
+
+  @author: Mohammad Abdo (@Jimmy-INL)
+
+  This class represents a base class for the validation algorithms
+  It inherits from the PostProcessor directly
+  ##TODO: Recast it once the new PostProcesso API gets in place
+"""
+
+#External Modules------------------------------------------------------------------------------------
+import numpy as np
+import xarray as xr
+#External Modules End--------------------------------------------------------------------------------
+
+#Internal Modules------------------------------------------------------------------------------------
+#from utils import xmlUtils
+from utils import InputData, InputTypes
+#import Files
+#import Distributions
+#import MetricDistributor
+from utils import utils
+from .. import ValidationBase
+# from utils.mathUtils import partialDerivative, derivatives
+#Internal Modules End--------------------------------------------------------------------------------
+
+class Representativity(ValidationBase):
+  """
+    Representativity is a base class for validation problems
+    It represents the base class for most validation problems
+  """
+
+  @classmethod
+  def getInputSpecification(cls):
+    """
+      Method to get a reference to a class that specifies the input data for
+      class cls.
+      @ In, cls, the class for which we are retrieving the specification
+      @ Out, specs, InputData.ParameterInput, class to use for
+        specifying input of cls.
+    """
+    specs = super(Representativity, cls).getInputSpecification()
+    parametersInput = InputData.parameterInputFactory("featureParameters", contentType=InputTypes.StringListType)
+    parametersInput.addParam("type", InputTypes.StringType)
+    specs.addSub(parametersInput)
+    targetParametersInput = InputData.parameterInputFactory("targetParameters", contentType=InputTypes.StringListType)
+    targetParametersInput.addParam("type", InputTypes.StringType)
+    specs.addSub(targetParametersInput)
+    targetPivotParameterInput = InputData.parameterInputFactory("targetPivotParameter", contentType=InputTypes.StringType)
+    specs.addSub(targetPivotParameterInput)
```

Comment:
please provide descriptions for these parameters using "descr" keyword

Reply:
Done